pkg/daemon/update.go

```diff
@@ -327,6 +327,12 @@ func (dn *Daemon) applyOSChanges(oldConfig, newConfig *mcfgv1.MachineConfig) (re
 
 	var osImageContentDir string
 	if mcDiff.osUpdate || mcDiff.extensions || mcDiff.kernelType {
+		// When we're going to apply an OS update, switch the block
+		// scheduler to BFQ to apply more fairness between etcd
+		// and the OS update.
+		if err := setRootDeviceSchedulerBFQ(); err != nil {
```

Comment:
What restores the original scheduler implementation following an update? A byproduct of restarting the node?

Reply:
Yep.